_episodes/08-ghpages-intro.md

```diff
@@ -4,12 +4,8 @@ teaching: 10
 exercises: 0
 questions:
 - "What are GitHub Pages?"
-- "How can I make a personal website with GitHub Pages?"
 objectives:
 - "Walk through the basics of GitHub Pages."
-- "Show the interplay between Jekyll and GH-pages to create your own website."
-keypoints:
-- "The same basics are applied to any template used with Jekyll/GH-pages."
 ---
```

Comment:
Think you need to remove this space.  YAML headers can be finicky. Might be why the keypoints are showing up at the bottom of this lesson right now.

Reply:
Though that might be because you no longer have any keypoints..not sure